hack/release/build/cross.sh

```diff
@@ -48,3 +48,6 @@ export GOOS=linux GOARCH=arm64
 export GOOS=linux GOARCH=ppc64le
 EOF
 )
+
+# add sha256 for binaries
+for f in ${REPO_ROOT}/bin/kind-*; do shasum -a 256 $f > $f.sha256sum; done
```

Comment:
the original output for `shasum` is something like `hashValue  fileName`, I think it only needs the hash value here?  

https://github.com/kubernetes/kompose/blob/0b331d9e5dede975d62ca382b2dd7dff89019f6e/script/release.sh#L243

Reply:
how do you know the filaname to check it with the command?

```
[aojea@aojea-laptop ~]$ shasum -a 256 ifac.go > ifac.sha
[aojea@aojea-laptop ~]$ sha256sum -c ifac.sha 
ifac.go: OK
[aojea@aojea-laptop ~]$ cp ifac.sha ifac.shaonly
[aojea@aojea-laptop ~]$ vi ifac.shaonly #removed the filename
[aojea@aojea-laptop ~]$ sha256sum -c ifac.shaonly 
sha256sum: ifac.shaonly: no properly formatted SHA256 checksum lines found
```

Reply:
You are right, it is not easy to check.  

however,I found that we have a lot of current tar files are only have the hash value,I'm just curious what is the reason for this.

`
➜  ~ gsutil cat gs://kubernetes-release/release/v1.21.0-alpha.0/kubernetes-test-windows-amd64.tar.gz.sha256   
e3b8a57d0bba4665f54874e4478bf2b360254314a482f4ba36f15ce228c561b7%     

➜  ~ gsutil cat gs://kubernetes-release/release/v1.21.0-alpha.0/kubernetes-src.tar.gz.sha256   
7eaf8fa1aa03afead4292d1e10dbc839439b4cd45a5a749bac242f8cf0040f45%    
`

Reply:
not an expert here, I always used the hash filename format, that is what shasum reports directly

Reply:
I have also used hash filename format for one of the CLI in the past. Not sure what benefit other approach has.